domains/measurements/chart/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: eo-measurements
 description: A chart containing the measurements domain.
 
 type: application
-version: 0.1.9
+version: 0.1.10
```

Comment:
I would argue these changes merit a major version change instead of a patch.

Reply:
bumped